### PR TITLE
5th level

### PR DESCRIPTION
**Pull request by mongoose11235813**
I made a new level for this game. Could you please pull it?

### CODE REVIEW

**Comment by janosgyerik** [comment]
Sounds awesome! Thanks for this, I'll take a look and probably pull tonight!

File: levels/05_theCorridor.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+}
+#END_PROPERTIES#
+/***************
+ * theCorridor.js *
+ ***************
+ */
+
+function startLevel(map) {
+#START_OF_START_LEVEL#
+    map.displayChapter('Chapter 4.1\nChapters are supposed to be more than one level long');
```
Comment (janosgyerik) [inline levels/05_theCorridor.jsx]: are they?

**Comment by janosgyerik** [comment]
I like this level, great work!

File: levels/05_theCorridor.jsx (hunk 1 above)
Comment (mongoose11235813) [inline levels/05_theCorridor.jsx:14]: Well. In the "untrusted" they are:
Chapter 1 Breakout: levels 1 to 7
Chapter 2 Raiders of the Lost Algorithm: levels 8 to 14
Chapter 3 Betrayal: levels 15 to 21

Comment (janosgyerik) [inline levels/05_theCorridor.jsx:14]: Oh I see, point taken!

**Comment by mongoose11235813** [comment]
BTW what is your solution for this level?
My solution:
https://gist.github.com/mongoose11235813/5b7f10470d368797d04a

**Comment by mongoose11235813** [comment]
> I like this level, great work!

Thanks

**Pull request merged**